File: tests/C#/sample_code.cs
using System;

class Program
{
    // Function to sum two numbers
    static double Sum(double a, double b)
    {
        return a + b;
    }

    // Main function that takes arguments from the command line
    static void Main(string[] args)
    {

        try
        {
            // Convert the arguments to numbers
            double num1 = Convert.ToDouble(args[1]);
            double num2 = Convert.ToDouble(args[2]);

            // Call the sum function
            double result = Sum(num1, num2);

            // Print the result
            Console.WriteLine("The sum is: " + result);
        }
        catch (FormatException)
        {
            Console.WriteLine("Invalid input. Please enter valid numbers.");
        }
    }
}

File: tests/Go/sample_code.go
package main

import (
	"fmt"
	"math"
	"os"
	"strconv"
)

// power calculates base^exp
func power(base float64, exp float64) float64 {
	return math.Pow(base, exp)
}

func main() {
	if len(os.Args) < 3 {
		fmt.Println("Usage: go run main.go <base> <exponent>")
		return
	}

	// Convert command-line arguments to float64
	base, err1 := strconv.ParseFloat(os.Args[1], 64)
	exp, err2 := strconv.ParseFloat(os.Args[2], 64)
	if err1 != nil || err2 != nil {
		fmt.Println("Error: Please provide valid numbers for base and exponent.")
		return
	}

	// Calculate and print result
	result := power(base, exp)
	fmt.Printf("%f^%f = %f\n", base, exp, result)
}
